WebInkEnhancement/explainer.md

```diff
@@ -126,10 +126,15 @@ dictionary PenStrokeStyle {
 }
 
 interface InkPresenter {
+    constructor(DOMString type, optional Element presentationArea);

```

Comment:
Instead of adding these constructors, you should just add a parameter to requestPresenter above